composer.json

```diff
@@ -19,6 +23,7 @@
     "wordpress/wordpress": "^5.2",
     "wp-cli/wp-cli-bundle": "^2.2",
     "wp-coding-standards/wpcs": "^2.2",
+    "wpackagist-plugin/advanced-custom-fields": "*",
```

Comment:
@kidunot89 forgive my popping in without context, but what is the reason for requiring ACF? This believe will install this plugin in cases someone manages Stream installations using Composer.

Reply:
:thinking: See as this is a **dev** dependency, I don't believe that is the case.

Reply:
@kidunot89 whoops. I did not expand the view and mistakenly thought this was under `require`. 😬  I will not install as require-dev is read a `root` only.